backend/substrapp/tasks/k8s_backend.py

```diff
@@ -651,7 +658,15 @@ def k8s_remove_local_volume(volume_id):
         spec=spec
     )
 
-    k8s_client.create_namespaced_pod(body=pod, namespace=NAMESPACE)
+    try:
+        k8s_client.create_namespaced_pod(body=pod, namespace=NAMESPACE)
+    except kubernetes.client.rest.ApiException as e:
+        if e.status == 409 and '"reason":"AlreadyExists"' in e.body:
+            logger.info(f'Cannot create pod {NAMESPACE}/{job_name}: Already exists. Skipping local volume deletion.')
+            return
```

Comment:
@inalgnu @Kelvin-M somehow the `clean-xxx` pod sometimes already exists. If that's the case, I suggest to just skip the whole function. Agree?

Reply:
@AurelienGasser yes agree, the pod name depends on the volume_id which depends on the compute plan id. For the same compute plan this could be skipped.